### PR TITLE
Test NFS export client addr option

### PR DESCRIPTION
**Pull request by msainiRedhat**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by msainiRedhat** [comment]
Logs - http://magna002.ceph.redhat.com/cephci-jenkins/msaini/nfs/client-addr/cepci-log/

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by MohitBis — DISMISSED** [review]
LGTM

File: tests/nfs/nfs_client_permission_export.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            pass
+        else:
+            log.error(f"Mount passed on unauthorized client: {clients[0].hostname}")
+
+    except Exception as e:
+        log.error(f"Failed to perform export client addr validation : {e}")
+        return 1
+    finally:
+        log.info("Cleaning up")
+        # Cleaning up the squash export and mount dir
```
Comment (pranavprakash20) [inline tests/nfs/nfs_client_permission_export.py:92]: is the comment correct? Its not the squash export that is created, please ignore if I got it wrong

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Review by vdas-redhat — APPROVED** [review]

**Comment by vdas-redhat** [comment]
Looks good to me

**Pull request merged**